modules/farm_fd2_examples/src/entrypoints/selector_base/App.vue

```diff
@@ -53,14 +53,26 @@
         </td>
       </tr>
       <tr>
-        <td>Include Add Button</td>
+        <td>PopupUrl</td>
         <td>
-          <BFormCheckbox
-            id="add-button-checkbox"
-            data-cy="add-button-checkbox"
-            switch
-            v-model="includeAddButton"
-          />
+          <BButton
+            id="popup-url-button"
+            data-cy="popup-url-button"
+            variant="outline-primary"
+            size="sm"
+            v-on:click="this.popupUrl = ''"
```

Comment:
I would suggest using `Not Null Url` or something here and in the button text.  As in my other comment an "Empty URL" or `''` seems somehow special and could be confusing to the reader who (like me) might wonder, "Why an empty URL?" when you just mean a `notNullUrl`